core/ir/instr_inline_api.h

```diff
@@ -175,6 +175,13 @@ opnd_is_predicate_zero(opnd_t op)
     return opnd_is_predicate_reg(op) && ((op.aux.flags & DR_OPND_IS_ZERO_PREDICATE) != 0);
 }
 
+INSTR_INLINE
+bool
+opnd_is_governing(opnd_t op)
+{
+    return opnd_is_predicate_reg(op) && ((op.aux.flags & DR_OPND_IS_GOVERNING) != 0);
```

Comment:
Because these can be used for clients, I believe TEST is undefined. The other test macros in this file all appear to be behind a ifded DYNAMORIO_INTERNAL